index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Lab 1</title>
+    <meta name="viewport" content="device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles/reset.css">
+    <link rel="stylesheet" href="styles/base.css">
+    <link rel="stylesheet" href="styles/layout.css">
+    <link rel="stylesheet" href="styles/modules.css">
+    <link rel="stylesheet" href="styles/theme.css">
+  </head>
+  <body>
+    <header>

```

Comment:
You could try refactoring so that you are using your header and footer tags. Otherwise, I would suggest removing them from your html since they don't have any content